CodeStarterCamp_Week3/main.swift

```diff
@@ -1,12 +1,63 @@
-//
-//  main.swift
-//  CodeStarterCamp_Week3
-//
-//  Created by yagom.
-//  Copyright © yagom academy. All rights reserved.
-//
 
 import Foundation
 
-print("Hello, World!")
+struct Person {
+    var name: String
+    var gender: String
+    var age: Int
+    var money: Int
+    
+    mutating func orderCoffee(coffeeShop: CoffeeShop, coffee: Coffee) {
+        if let price = coffeeShop.menu[coffee] {
+            if money >= price {
+                money = money - price
+                print("커피 가격은 \(price)원이고 남은 돈 \(money)원 입니다.")
+            } else {
+                print("돈이 부족하여 커피를 구매할 수 없습니다.")
+            }
+            
+        }
+    }
+}
 
+
+class CoffeeShop {
+    var shopName: String
+    var revenue: Int?
+    var barista: Person
+    var menu: [Coffee: Int] = [:]
+    var pickUpTable: [String] = []
+    
+    func makeCoffee(ordererName: String, coffee: Coffee) {
+        pickUpTable.append("\(coffee)")
+        for num in 0...(pickUpTable.count - 1) {
+            print("\(ordererName)님 주문하신 \(pickUpTable[num]) 나왔습니다.")
+            pickUpTable.removeAll()
+        }
+    }
+    
+    init(shopName: String, barista: Person, pickUpTable: [String]) {
+        self.shopName = shopName
+        self.barista = barista
+        self.pickUpTable = pickUpTable
+        for coffee in Coffee.all {
+            menu[coffee] = coffee.rawValue
+        }
+    }
```

Comment:
```suggestion
    init(shopName: String, barista: Person, pickUpTable: [String]) {
        self.shopName = shopName
        self.barista = barista
        self.pickUpTable = pickUpTable
        
        for coffee in Coffee.all {
            menu[coffee] = coffee.rawValue
        }
    }
```
개행이 있으면 좋겠네요 🙂

일반적으로 `init`은 프로퍼티 다음에 위치합니다!